src/murmur/ServerUser.h

```diff
@@ -108,6 +109,7 @@ class ServerUser : public Connection, public User {
 public:
 	enum State { Connected, Authenticated };
 	State sState;
+	ClientType client_type;
```

Comment:
```suggestion
	ClientType m_clientType;
```

Reply:
I replaced ```client_type``` with ```m_clientType``` everywhere except ```Mumble.proto```, replaced it with ```m_client_type``` there